src/procedure/nodes/node.cpp

```diff
@@ -101,25 +102,17 @@ ProcedureNode::NodeType ProcedureNode::type() const { return type_; }
 // Return whether the supplied context is relevant for the current node
 bool ProcedureNode::isContextRelevant(NodeContext targetContext) const
 {
-    // If the node is suitable in Any context return immediately
-    if (std::find(relevantContexts_.begin(), relevantContexts_.end(), ProcedureNode::NodeContext::AnyContext) !=
-        relevantContexts_.end())
+    // Check for spurious context checks
+    if (targetContext == ProcedureNode::NodeContext::InheritContext)
+        throw(std::runtime_error(fmt::format("Attempted to context check node '{}' with an InheritedContext.\n")));
```

Comment:
```suggestion
        throw(std::runtime_error(fmt::format("Attempted to context check node '{}' with an InheritedContext.\n", !name_.empty() ? name_ : nodeTypes().keyword(type_))));
```

Missing argument here. Although I'm not 100% convinced by this check, since when I add an `IfValueInRange` node and then try and add another node in the branch, this gets thrown.

Reply:
Interesting.   I was only testing via an input file, not the GUi, so obviously missed something here!